src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/categories/CategorySyncIT.java

```diff
@@ -79,19 +77,21 @@ void setupTest() {
     callBackErrorResponses = new ArrayList<>();
     callBackExceptions = new ArrayList<>();
     callBackWarningResponses = new ArrayList<>();
+    categorySync = new CategorySync(buildCategorySyncOptions(50));
```

Comment:
CategorySyncOptionsBuilder has a default batch size 50. I think we don't need to intentionally set it in Integration Test case.

Reply:
buildCategorySyncOptions is a method and used to pass different batch size, here default is passed, it's fluent so I can not do much about it.